### PR TITLE
boot: add page

### PR DESCRIPTION
**Pull request by jedahan**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/boot.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# boot
+
+> Build tooling for Clojure.
```
Comment (agnivade) [inline pages/common/boot.md]: Perhaps - `for the Clojure programming language` makes it more clear.

File: pages/common/boot.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Generate scaffolding for a new project based on a template:
+
+`boot -d boot/new new --template {{template_name}} --name {{project_name}}`
```
Comment (agnivade) [inline pages/common/boot.md]: Is there an expanded form of `-d` ? Because the other options use the expanded form.

File: pages/common/boot.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Start a REPL session either with the project or standalone:
+
+`boot repl`
+
+- Build for development (if using the boot/new template):
```
Comment (agnivade) [inline pages/common/boot.md]: How would one build if _not_ using the boot/new template ?

Comment (jedahan) [reply pages/common/boot.md]: The "uberjar" as far as I understand is the canonical way. I moved it to above the other ways that are part of the recommended templated ways.

**Comment by agnivade** [comment]
ping @sbrl

**Comment by sbrl** [comment]
Oops! I think this must have slipped through the net :P

Yep, looks ok to me! Thanks, @jedahan :D

**Pull request merged**